lib/tasks/test.js

```diff
@@ -60,17 +82,115 @@ module.exports.nonSilentCompilation = function(gulp) {
 };
 
 module.exports.npmTest = function() {
-	var packageJson = files.getPackageJson();
-	if (packageJson && packageJson.scripts && packageJson.scripts.test) {
-		commandLine.run('npm', ['test'])
-			.then(function(output) {
-				log.primary('Running "npm test"...');
-				log.secondary(output.stdout);
-			}, function(output) {
-				log.primary('Running "npm test"...');
-				log.secondaryError(output.stderr);
+	return new Promise(function() {
```

Comment:
I'd suggest resolving or rejecting this promise.  Seems weird to return a Promise that will never resolve.